content/docs/0.15.x/reference/files/distributor.yaml/index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: distributor.yaml
+description: Define a distributor
+weight: 125
+---
+
+You can create your own distributor by writing a dedicated distributor deployment yaml:
+
+## Syntax
+
+```yaml
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: some-service-monitoring-configure-distributor
+  namespace: keptn
+spec:
+  selector:
+    matchLabels:
+      run: distributor
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        run: distributor
+    spec:
+      containers:
+        - name: distributor
+          image: keptndev/distributor:latest
```

Comment:
let's refer to the latest official 0.15.x image keptn/distributor:0.15.0 here

Reply:
done